package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "notificationapi-node-server-sdk",
-  "version": "0.17.0",
+  "version": "0.18.0",

```

Comment:
0.17.1 would be more appropriate since there have been no change in the expected behavior of the code.